internal/handler/msg_currentop.go

```diff
@@ -39,8 +39,44 @@ func (h *Handler) MsgCurrentOp(connCtx context.Context, msg *wire.OpMsg) (*wire.
 		return nil, err
 	}
 
+	// TODO https://github.com/FerretDB/FerretDB-DocumentDB/issues/78
+	doc, err := spec.Decode()
+	if err != nil {
+		return nil, lazyerrors.Error(err)
+	}
+
+	dbName, err := getRequiredParam[string](doc, "$db")
+	if err != nil {
+		return nil, err
+	}
+
+	currentOpSpec := must.NotFail(wirebson.MustDocument(
+		"aggregate", int32(1),
+		"pipeline", wirebson.MustArray(wirebson.MustDocument("$currentOp", wirebson.MustDocument())),
+		// use large batchSize to get all results in one batch
+		"cursor", wirebson.MustDocument("batchSize", int32(10000)),
+	).Encode())
+
+	page, cursorID, err := h.Pool.Aggregate(connCtx, dbName, currentOpSpec)
```

Comment:
Why don't we use https://github.com/FerretDB/FerretDB/blob/5a144ed783fc692fb0145b5eede6b890471fe8d1/internal/documentdb/documentdb_api/documentdb_api.go#L171-L174 ?

Reply:
All tests pass with it